### PR TITLE
[plugin.video.svtplay] 5.0.1

### PR DESCRIPTION
**Pull request by linqcan**
### Description

Updating plugin.video.svtplay. This version is Py2/3 compatible.
Dropping support for Kodi <17

Let me know if you spot any issues!
### Checklist:

- [x ] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x ] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x ] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.
- If you see no activity on your PR after a week (so at least one weekend has passed) then please go to the #kodi-dev freenode IRC channel to reach out to the team

### CODE REVIEW

**Comment by linqcan** [comment]
I'm on it Travis 😎

**Comment by linqcan** [comment]
Travis is happy now.

**Comment by enen92** [comment]
Hey @linqan. Can you mark the version in gotham as broken as you specifically state you removed the support to lower versions? Please do it in a different PR.

**Comment by linqcan** [comment]
@enen92 Sure, can we merge this one first though?

**Comment by enen92** [comment]
Sure, but please open the PR . We'll make sure this one goes in first.

**Comment by linqcan** [comment]
> Sure, but please open the PR . We'll make sure this one goes in first.

#2520 is now ready.

File: plugin.video.svtplay/addon.xml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  </extension>
+  <extension point="xbmc.addon.metadata">
+    <language>sv</language>
+    <summary lang="en_GB">Watch content from SVT Play.</summary>
+    <summary lang="sv_SE">Titta på innehåll från SVT Play.</summary>
+    <description lang="en_GB">With this addon you can stream content from SVT Play (svtplay.se).</description>
+    <description lang="sv_SE">Med denna addon kan du strömma innehåll från SVT Play (svtplay.se).</description>
+    <disclaimer lang="en_GB">This is an unofficial addon and it is not supported or maintained by SVT. Some parts of this addon may not be legal in your country of residence - please check with your local laws before installing.</disclaimer>
+    <platform>all</platform>
+    <license> GNU GENERAL PUBLIC LICENSE. Version 3, 29 June 2007</license>
```
Comment (enen92) [inline plugin.video.svtplay/addon.xml]: GPL-3.0 if possible (spdx identifier)

File: plugin.video.svtplay/resources/lib/logging.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+from __future__ import absolute_import,unicode_literals
+import xbmc # pylint: disable=import-error
+
+def log(msg, loglevel=xbmc.LOGDEBUG):
+    log_str = "[plugin.video.svtplay] {}".format(msg)
+    try:
```
Comment (enen92) [inline plugin.video.svtplay/resources/lib/logging.py]: In python3 you can now use fstrings :)

File: plugin.video.svtplay/default.py [hunk 3]
```diff
@@ -0,0 +1,308 @@
...
+    return
+  if show_obj["videoUrl"]:
+    play_video(show_obj)
+  else:
+    dialog = xbmcgui.Dialog()
+    dialog.ok("SVT Play", localize(30100))
+
+def play_video(show_obj):
+  player = xbmc.Player()
+  start_time = time.time()
```
Comment (enen92) [inline plugin.video.svtplay/default.py]: Our API could well be improve to allow setting this without waiting for the player to start

File: plugin.video.svtplay/addon.xml [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    <website>https://github.com/nilzen/xbmc-svtplay</website>
```
Comment (Rechi) [inline plugin.video.svtplay/addon.xml]: point to the content providers website

File: plugin.video.svtplay/resources/lib/logging.py (hunk 2 above)
Comment (linqcan) [reply plugin.video.svtplay/resources/lib/logging.py]: Yeah, but are those Py2 compatible? 🤔 @enen92

File: plugin.video.svtplay/addon.xml (hunk 4 above)
Comment (linqcan) [reply plugin.video.svtplay/addon.xml]: That's not how I interpret the documentation: https://kodi.wiki/view/Addon.xml#.3Cwebsite.3E
"""
The <website> element provides the website URL for this specific add-on.
<website>http://www.myaddonwebsite.com/</website>
"""

File: plugin.video.svtplay/default.py (hunk 3 above)
Comment (linqcan) [reply plugin.video.svtplay/default.py]: I honestly don't know why this is needed, I inherited the code from the orignal developer.

But if this is something all video addon devs have to do then I agree with you that the API should be updated to simplify this action.

File: plugin.video.svtplay/addon.xml (hunk 1 above)
Comment (linqcan) [reply plugin.video.svtplay/addon.xml]: Turns out the SPDX is "GPL-3.0-only". I'll update to reflect that.

File: plugin.video.svtplay/resources/lib/logging.py (hunk 2 above)
Comment (enen92) [reply plugin.video.svtplay/resources/lib/logging.py]: Nop :) it was just a tip

File: plugin.video.svtplay/addon.xml (hunk 4 above)
Comment (enen92) [reply plugin.video.svtplay/addon.xml]: Thanks for pointing out the wiki could be improved. We'll have that changed. Website refers to the content website or in some cases to the add-on website (eg for any addon that provides content from multiple websites or that have a website dedicated to the add-on). Usually the source element already points to the upstream code so there's no need to repeat it. Can you have this changed for the PRs to be merged? Thanks

Comment (linqcan) [reply plugin.video.svtplay/addon.xml]: I have changed this now.

File: plugin.video.svtplay/resources/lib/logging.py (hunk 2 above)
Comment (linqcan) [reply plugin.video.svtplay/resources/lib/logging.py]: Ah, OK. Thanks! 👍

**Comment by linqcan** [comment]
This and #2520 should be OK now. Thanks for all the feedback!

**Pull request merged**